python/RDS_ENHANCED_MONITORING_ENABLED/RDS_ENHANCED_MONITORING_ENABLED.py

```diff
@@ -0,0 +1,395 @@
+#####################################
+
+Description:
+  Checks whether enhanced monitoring is enabled for Amazon RDS instances.
+Trigger:
+  Configuration Changes
+Reports on:
+  AWS::RDS::DBInstance
+Rule Parameters:
+ monitoringInterval
+ (Optional) An integer value in seconds between points when Enhanced Monitoring metrics are collected for the DB instance. Valid Values are 1, 5, 10, 15, 30, 60.
+
+Scenarios:
+  Scenario: 1
+     Given: Value for the rule parameter 'monitoringInterval' is invalid.
+      Then: Return ERROR
+  Scenario: 2
+     Given: 'monitoringInterval' is '0' in configuration item of the Amazon RDS instance.
+      Then: Return NON_COMPLIANT
+  Scenario: 3
+     Given: Value for the rule parameter 'monitoringInterval' is provided and is valid
+       And: 'monitoringInterval' in configuration item of the Amazon RDS instance does not match the rule parameter value
+      Then: Return NON_COMPLIANT
+  Scenario: 4
+     Given: Value for the rule parameter 'monitoringInterval' is not provided
+       And: 'monitoringInterval' is a non-zero value in configuration item of the Amazon RDS instance.
+      Then: Return COMPLIANT
+  Scenario: 5
+     Given: Value for the rule parameter 'monitoringInterval' is provided and is valid
+       And: 'monitoringInterval' in configuration item of the Amazon RDS instance matches the rule parameter value
+      Then: Return COMPLIANT
+
+'''
+
+import json
+import sys
+import datetime
+import boto3
+import botocore
+
+try:
+    import liblogging
+except ImportError:
+    pass
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::RDS::DBInstance'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+# Other parameters (no change needed)
+CONFIG_ROLE_TIMEOUT_SECONDS = 900
+
+#############
+# Main Code #
+#############
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    if int(configuration_item['configuration']['monitoringInterval']) == 0:
+        return build_evaluation_from_config_item(configuration_item, 'NON_COMPLIANT', annotation="Enhanced Monitoring interval for this Amazon RDS instance is not configured.")
+
+    if not valid_rule_parameters and (int(configuration_item['configuration']['monitoringInterval']) > 0):
+        return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
+
+    if valid_rule_parameters and int(configuration_item['configuration']['monitoringInterval']) != int(valid_rule_parameters['monitoringInterval']):
+        return build_evaluation_from_config_item(configuration_item, 'NON_COMPLIANT', annotation="Enhanced Monitoring interval for this Amazon RDS instance is not set with period:" + valid_rule_parameters['monitoringInterval'])
+
+    return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')

```

Comment:
I don't like the idea of returning COMPLIANT as a default return just in case we missed something. Maybe we can run checks and return COMPLIANT only if certain conditions are met.
Return NON_COMPLIANT as default.

@jongogogo let me know your thoughts on this.